### ISSUE TITLE
[Mac] Some deadkey combinations are not working

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#1716]
**Describe the bug**
Some deadkey combinations are not working on Keyman for macOS.

**To Reproduce**

Per [Steve White](https://community.software.sil.org/t/ring-diacritic-indic-roman-transliteration-not-working/2019/4?u=marc_keyman): I’m trying out SIL_Indic_Roman on a Mac and I see what kalibhakta describes. The 0 + r sequence does not do anything in Pages or Textedit.

Most of the keyboard does work. Everything in the first table of the documentation is working: á ‘a á â à ä ā ṅ ṭ

For the second table “Combinations common to Bangla/Assamese” nothing works, except the m̐

OS version 10.14.3 (Mojave). Keyman version 11.0.220.

---

(Delete the irrelevant sections)

**Keyman for macOS**
 - OS: macOS Mojave
 - Keyman version: 11.0.220
 - Target Application: TextEdit, Pages, anything else

---
 
**Keyboard**
 - Keyboard name: [sil_indic_roman](https://keyman.com/keyboards/sil_indic_roman)
 - Keyboard version: 1.3

**Comment by mcdurdin** [comment]
Steve White notes:

It looks like defining more than 9 deadkeys in a KMN file will not work for Mac OS. Deadkey #10 fails to match any later output.

See this sample file. Deadkeys 1-9 work fine, 10 fails.

```
store(&TARGETS) 'windows macosx'
begin Unicode > use(main)

group(main) using keys
+ '1' > deadkey(1)
+ '2' > deadkey(2)
+ '3' > deadkey(3)
+ '4' > deadkey(4)
+ '5' > deadkey(5)
+ '6' > deadkey(6)
+ '7' > deadkey(7)
+ '8' > deadkey(8)
+ '9' > deadkey(9)
+ '0' > deadkey(10)

deadkey(1) + '#' > '0001'
deadkey(2) + '#' > '0010'
deadkey(3) + '#' > '0011'
deadkey(4) + '#' > '0100'
deadkey(5) + '#' > '0101'
deadkey(6) + '#' > '0110'
deadkey(7) + '#' > '0111'
deadkey(8) + '#' > '1000'
deadkey(9) + '#' > '1001' 
deadkey(10) + '#' > '1010'
```

**Comment by mcdurdin** [comment]
Fixed by #1839
